File: Day-03-Working with Arrays/04-Remove-Duplicates/Solution.py
n = int(input())
arr = list(map(int, input().split()))
k = 1
for i in range(1, n):
    if arr[i] != arr[k-1]:
        arr[k] = arr[i]
        k += 1

for i in range(n):
    if i < k:
        print(arr[i], end=" ")
    else:
        print("_", end=" ")

File: Day-03-Working with Arrays/04-Remove-Duplicates/Solution.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    int arr[n];
    for(int i = 0; i < n; i++)
        cin >> arr[i];

    int k = 1;

    for(int i = 1; i < n; i++) {
        if(arr[i] != arr[k-1]) {
            arr[k] = arr[i];
            k++;
        }
    }

    for(int i = 0; i < n; i++) {
        if(i < k)
            cout << arr[i] << " ";
        else
            cout << "_ ";
    }
}